### PR TITLE
Integrating Microsoft Teams

### PR DESCRIPTION
**Pull request by Andrioshe**
Add support to push notifications to Microsoft Teams.

I am sure that the code is not good but I am looking forward to improve it.

### CODE REVIEW

**Comment by Heisenberg74** [comment]
@Andrioshe I am really interested in your PR. Could you please add comments on types Section and Fact in order to pass all checkpoints?

Thank you very much.

File: internal/notif/teams/teams.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		},
+	}
+}
+
+// Name returns notifier's name
+func (c *Client) Name() string {
+	return "teams"
+}
+
+type Sections struct {
```
Comment (Heisenberg74) [inline internal/notif/teams/teams.go]: Could you please add comment here ? @Andrioshe

File: internal/notif/teams/teams.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	ActivityTitle    string `json:"activityTitle"`
+	ActivitySubtitle string `json:"activitySubtitle"`
+	Facts            []Fact `json:"facts"`
+}
+
+type Fact struct {
```
Comment (Heisenberg74) [inline internal/notif/teams/teams.go]: Could you please add comment here ? @Andrioshe

File: internal/notif/teams/teams.go (hunk 1 above)
Comment (Andrioshe) [reply internal/notif/teams/teams.go]: done 👍

File: internal/notif/teams/teams.go (hunk 2 above)
Comment (Andrioshe) [reply internal/notif/teams/teams.go]: done 👍

**Comment by Heisenberg74** [comment]
Great ! Hope it will be accepted 🙂

**Comment by crazy-max** [comment]
Superseded #72

**Pull request closed without merging**